dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/ValueType.java

```diff
@@ -46,31 +46,31 @@
 @JacksonXmlRootElement( localName = "valueType", namespace = DxfNamespaces.DXF_2_0 )
 public enum ValueType
 {
-    TEXT( String.class, true ),
-    LONG_TEXT( String.class, true ),
-    LETTER( String.class, true ),
-    PHONE_NUMBER( String.class, false ),
-    EMAIL( String.class, false ),
-    BOOLEAN( Boolean.class, true ),
-    TRUE_ONLY( Boolean.class, true ),
-    DATE( LocalDate.class, false ),
-    DATETIME( LocalDateTime.class, false ),
-    TIME( String.class, false ),
-    NUMBER( Double.class, true ),
-    UNIT_INTERVAL( Double.class, true ),
-    PERCENTAGE( Double.class, true ),
-    INTEGER( Integer.class, true ),
-    INTEGER_POSITIVE( Integer.class, true ),
-    INTEGER_NEGATIVE( Integer.class, true ),
-    INTEGER_ZERO_OR_POSITIVE( Integer.class, true ),
-    TRACKER_ASSOCIATE( TrackedEntityInstance.class, false ),
-    USERNAME( String.class, false ),
-    COORDINATE( Point.class, true ),
-    ORGANISATION_UNIT( OrganisationUnit.class, false ),
-    AGE( Date.class, false ),
-    URL( String.class, false ),
-    FILE_RESOURCE( String.class, false ),
-    IMAGE( String.class, false );
+    TEXT( String.class, true, null ),
```

Comment:
Maybe it makes sense to add another constructor her? so we don't have to add null for the options in all these cases